core/src/validator.rs

```diff
@@ -1366,6 +1366,8 @@ impl Validator {
             ("id", id.to_string(), String),
             ("version", solana_version::version!(), String),
             ("cluster_type", genesis_config.cluster_type as u32, i64),
+            ("waited_for_supermajority", waited_for_supermajority, bool),
+            ("expected_shred_version", config.expected_shred_version, Option<i64>),
```

Comment:
how does influx handle options? :thinking:

Reply:
checked some of other metrics that use options, it converts None to whatever null representation it uses.

Reply:
_interesting..._ i guess that means `None`s are indiscernible from datapoints not existing given that the chronograf queries typically `FILL(null)` 🤔

seems like if that's a problem, it's problem for another pr. there is already extensive use of this pattern